actor/src/main/java/org/apache/pekko/japi/JAPI.java

```diff
@@ -15,8 +15,11 @@
 
 import scala.collection.Seq;
 
+@Deprecated
```

Comment:
could you you use a format like this?

```
    /**
     * @deprecated This method is no longer acceptable to compute time between versions. Use {@link Utils#calculatePeriod(Machine)} instead.
     */
    @Deprecated(since = "1.1.0")
```

The text is grabbed from another example but the idea is that we need to specify what the recommended alternative is.

Reply:
Since will require java 9，but I can update it